dockers/docker-orchagent/ipinip.json.j2

```diff
@@ -56,15 +55,24 @@
             "ttl_mode":"pipe"
         },
         "OP": "SET"
-    }
+    }{% if ipv4_addresses %},
 {% endif %}
-{% if ipv4_loopback_addresses and ipv6_loopback_addresses %}    ,
+{% for prefix in ipv4_addresses|sort %}
+    {
+        "TUNNEL_DECAP_TERM_TABLE:IPINIP_TUNNEL:{{ prefix | ip }}" : {
+            "term_type":"P2MP"
+        },
+        "OP": "SET"
+    }{% if not loop.last %},
+{% endif %}
+{% endfor %}
+{% endif %}
+{% if ipv4_loopback_addresses and ipv6_loopback_addresses %},
```

Comment:
Why `ipv4_loopback_addresses` needs to be checked here?

Reply:
Let me list out all possible conditions:
if `ipv4_loopback_addresses` is empty, `ipv6_loopback_addresses` is empty: no tunnel && decap terms will be created, no need to add `,`.
if `ipv4_loopback_addresses` is not empty, `ipv6_loopback_addresses` is empty: only IPv4 tunnel && decap terms will be created, no need to add `,`.
if `ipv4_loopback_addresses` is empty, `ipv6_loopback_addresses` is not empty: only IPv6 tunnel && decap terms will be created, no need to add `,`.
if `ipv4_loopback_addresses` is not empty, `ipv6_loopback_addresses` is not empty: both IPv4&&IPv6 tunnel && decap terms will be created, ',' will be added to inter-connect those two parts.